pinot-common/src/main/java/org/apache/pinot/common/config/provider/TableCache.java

```diff
@@ -76,7 +76,7 @@ public class TableCache implements PinotConfigProvider {
   private final Set<SchemaChangeListener> _schemaChangeListeners = new HashSet<>();
 
   private final ZkHelixPropertyStore<ZNRecord> _propertyStore;
-  private final boolean _caseInsensitive;
+  private final boolean _ignoreCase;
```

Comment:
@Jackie-Jiang  I renamed the `caseInSensitive` everywhere to `ignoreCase` because I already got confused during my coding. Negating in naming will cause more trouble.